lib/utils/utils.js

```diff
@@ -160,6 +160,10 @@ function downloadFile(url, dest, cb) {
   const o_fs = require('fs-extra');
   var file = o_fs.createWriteStream(dest);
   (url.substring(0, 5) === 'https' ? https : http).get(url, function (response) {
+    if (response.statusCode !== 200) {
+      if (cb) cb(`Download failed, response code ${response.statusCode}`);
```

Comment:
Callback should always be defined

Reply:
Revised so that it's assumed callback is always supplied.